src/migration/1578909700398-Initialize.ts

```diff
@@ -1,16 +1,17 @@
 import {MigrationInterface, QueryRunner} from "typeorm";
 
-export class Initialize1578482938987 implements MigrationInterface {
-    name = 'Initialize1578482938987'
+export class Initialize1578909700398 implements MigrationInterface {
+    name = 'Initialize1578909700398'
 
     public async up(queryRunner: QueryRunner): Promise<any> {
         await queryRunner.query(`CREATE TABLE "activities" ("id" SERIAL NOT NULL, "type" character varying NOT NULL, CONSTRAINT "UQ_704a5fe2080d400189b76938cd1" UNIQUE ("type"), CONSTRAINT "PK_7f4004429f731ffb9c88eb486a8" PRIMARY KEY ("id"))`, undefined);
         await queryRunner.query(`CREATE TABLE "location" ("id" SERIAL NOT NULL, "venue" character varying NOT NULL, "street" character varying NOT NULL, "city" character varying NOT NULL, "country" character varying NOT NULL, CONSTRAINT "PK_876d7bdba03c72251ec4c2dc827" PRIMARY KEY ("id"))`, undefined);
         await queryRunner.query(`CREATE TABLE "perks" ("id" SERIAL NOT NULL, "type" character varying NOT NULL, CONSTRAINT "UQ_451e535e90c45489610ae107e26" UNIQUE ("type"), CONSTRAINT "PK_7858d2e37891967413d2010a278" PRIMARY KEY ("id"))`, undefined);
-        await queryRunner.query(`CREATE TABLE "speakers" ("id" SERIAL NOT NULL, "first_name" character varying NOT NULL, "last_name" character varying NOT NULL, "gender" character varying(1) NOT NULL, "bio" character varying, "job_title" character varying NOT NULL, "image" bytea NOT NULL, CONSTRAINT "PK_b3818c94af77a0cf73403ecef14" PRIMARY KEY ("id"))`, undefined);
+        await queryRunner.query(`CREATE TABLE "speakers" ("id" SERIAL NOT NULL, "first_name" character varying NOT NULL, "last_name" character varying NOT NULL, "gender" character varying(1) NOT NULL, "bio" character varying, "job_title" character varying, "image" bytea, CONSTRAINT "PK_b3818c94af77a0cf73403ecef14" PRIMARY KEY ("id"))`, undefined);
```

Comment:
it's possible we don't know or the speaker doesn't have a job, better to make `job_title` nullable

Reply:
cool